src/content/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager.mdx

```diff
@@ -340,31 +340,22 @@ Below are the applicable Docker or Kubernetes instructions to start the job mana
     title={<><img src={syntheticDocker} title="Docker icon" alt="Docker icon" style={{ height: '35px', width: '40px' }}/> Docker start procedure</>}
   >
     1. Locate your [private location key](#private-location-key).
-    2. Ensure you've enabled [Docker dependencies](#sandboxing-and-docker-deps) for sandboxing and [installed](#install-update) synthetics job manager on your system.
-    3. Run the appropriate script for your system. Tailor the common default `/var/run/docker.sock` in the following examples to match your system.
+    2. Ensure you've enabled [Docker dependencies](#sandboxing-and-docker-deps) for sandboxing and [installed Docker Engine](https://docs.docker.com/engine/install/) for your platform. If running as a [non-root user](#run-non-root), also complete the Docker Post-installation steps to [Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
+    3. Run the following command from a Linux Terminal. Tailor the common default for `/var/run/docker.sock` to match your system. For Windows, install a Linux distribution in WSL 2 and run the command from there.
 
-       * Linux/macOS:
-
```

Comment:
Do you want the gap between os and the docker run cmd? Linux bullet point has a gap windows doesn't. Not sure if this matters on the rendered page.

Reply:
I removed the bullet points completely. See changes on right.